.github/workflows/test.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '>=1.18.0'
```

Comment:
Reduce maintenance for us and users, and have the version pulled from the go.mod file: https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file
So for this it's probably `go-version-file: 'go.mod'`

Reply:
Good catch, thanks!